src/fondue/Fondue.js

```diff
@@ -20,20 +20,37 @@ export default class Fondue {
 	}
 
 	get isColor() {
-		return this.colorFormats.length >= 1;
+		return this.colorFormats.length > 0;
+	}
+
+	get hasFeatures() {
+		return (
+			Object.keys(this.features.fixed).length > 0 ||
+			Object.keys(this.features.optional).length > 0
+		);
+	}
+
+	get hasLanguages() {
+		return Object.keys(this.languageSystems).length > 0;
 	}
 
 	// Return an object of all language systems supported by
 	// either GSUB or GPOS. Tags are stripped ("ROM " → "ROM").
 	get languageSystems() {
 		const getLangs = (table) => {
-			return table
-				.getSupportedScripts()
-				.reduce((acc, script) => {
-					const scriptTable = table.getScriptTable(script);
-					return acc.concat(table.getSupportedLangSys(scriptTable));
-				}, [])
-				.map((lang) => lang.trim());
+			if (table) {
```

Comment:
It seems like this check could be moved to the outer function, as guard clause? I.e.:

```js
const gsubTable = this._font.opentype.tables.GSUB;
if(! gsubTable) return [];

const gsubLangs = getLangs(gsubTable);
...

Makes the nested function more readable and in general I think it's good to abort early.
```

Reply:
With the new code I'm still catching it in the code that gets the raw features. Please see if this is okay!